webpack.prod.config.js

```diff
@@ -35,6 +35,7 @@ const config = merge(baseConfig, {
       new OptimizeCSSAssetsPlugin({}),
       new TerserPlugin({
         terserOptions: {
+          keep_fnames: process.env.NODE_ENV === "production" ? false : /(GraphQLScalarType|GraphQLObjectType)/
```

Comment:
can you please add a comment in the code explaining why this is necessary?

Reply:
@cbrzn   In the latest dev branch we no longer use "staging", so once we merge the latest dev up to this branch, can we remove this line?